common/src/main/resources/db/changelog/v001/ab2d-1069-recreate-coverage-table.sql

```diff
@@ -0,0 +1,35 @@
+--liquibase formatted sql
+--  -------------------------------------------------------------------------------------------------------------------
+
+--changeset spathiyil:ab2d-1069-drop-coverage-table failOnError:true
+DROP TABLE coverage;
+
+--rollback CREATE TABLE coverage
+--rollback (
+--rollback     beneficiary_id      BIGINT              NOT NULL,
+--rollback     contract_id         BIGINT              NOT NULL
+--rollback );
+--rollback ALTER TABLE coverage ADD CONSTRAINT "fk_coverage_to_beneficiary" FOREIGN KEY (beneficiary_id) REFERENCES beneficiary (id);
+--rollback ALTER TABLE coverage ADD CONSTRAINT "fk_coverage_to_contract" FOREIGN KEY (contract_id) REFERENCES contract (id);
+--rollback ALTER TABLE coverage ADD CONSTRAINT "uc_coverage_beneficiary_id_contract_id" UNIQUE (beneficiary_id, contract_id);
+
+--  -------------------------------------------------------------------------------------------------------------------
+
+--changeset spathiyil:ab2d-1069-recreate-coverage-table-with-id-and-month failOnError:true
+CREATE TABLE coverage
+(
+    id                  BIGINT              NOT NULL,
+    contract_id         BIGINT              NOT NULL,
+    beneficiary_id      BIGINT              NOT NULL,
+    month               INTEGER             NOT NULL
```

Comment:
Turns out `month` is a keyword in SQL. 
I do not know if there is any danger in using the column name as `month`.
I could rename it to something like `active_month` unless there is a better name?

Reply:
`part_d_month`?

Reply:
done